### PR TITLE
Small deco note

### PR DESCRIPTION
**Pull request by gnizla**

### CODE REVIEW

File: OxfordBodleian/Juel-Jensen/BDLaethf24.xml [hunk 1]
```diff
@@ -162,9 +162,22 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                             </handNote>
                         </handDesc>
 
-                       <!-- <decoDesc>
+                       <decoDesc>
+                         <summary>Decoration on <locus target="#3r"/> could be contemporary with the making of the manuscript. That on <locus target="#4r"/>
+                       appears to be posterior.</summary>
+                         <decoNote type="ornamentation" xml:id="d1">
+                             <locus target="#3r"></locus>
+                             <desc>Monochrome unframed irregular interlace pattern painted in red and above the upper ruling margin and outlined with black ink.
```
Comment (thea-m) [inline OxfordBodleian/Juel-Jensen/BDLaethf24.xml:170]: I don't understand the sentence well, "painted in red and above" - is "and" here necessary, or is there perhaps a word missing?

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you for all the updates!

File: OxfordBodleian/Juel-Jensen/BDLaethf24.xml (hunk 1 above)
Comment (gnizla) [reply OxfordBodleian/Juel-Jensen/BDLaethf24.xml:170]: No the subject "interlace pattern" is "painted in red" and "painted above the upper ruling margin"  I could delete the "and" to make it clearer :)

Comment (thea-m) [reply OxfordBodleian/Juel-Jensen/BDLaethf24.xml:170]: Ah, now I get it!

**Review by thea-m — APPROVED** [review]

**Pull request merged**